### PR TITLE
Update Ruby SDK

### PR DESCRIPTION
**Pull request by biglovisa**
- `photos` field to `projects` 
- `average_price_per_tonne` field to `projects`
- `remaining_mass_g` field to `projects`
- `standard` field to `projects`
- validations on `mass_g` field (has to be greater than 1 and less than 2,000,000,000).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
+## [1.2.0] - 2020-09-17
+
+### Added
+
+- `photos` optional field to `projects` creation
+- `average_price_per_tonne` field to `projects`
+- `remaining_mass_g` field to `projects`
+- `standard` field to `projects`
+- validations on `mass_g` field (has to be greater than 1 and less than 2,000,000,000).
```
Comment (thdaraujo) [inline CHANGELOG.md:18]:
```suggestion
- validations on `mass_g` field (has to be greater than 1 and less than 2,000,000,000) to `orders` and `estimates` creation
```

maybe add a little bit more context

**Review by thdaraujo — DISMISSED** [review]
thanks for updating the gem! 

do you mind adding integration tests to these new fields?
please also update the readme examples.

**Comment by biglovisa** [comment]
@thdaraujo good point! Just pushed a commit with integration tests.

File: spec/models/standard_spec.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
+=begin
```
Comment (bspellacy) [inline spec/models/standard_spec.rb:1]: Let's just drop this spec. It doesn't really test anything.

File: spec/models/photo_spec.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
+=begin
+#Patch API V1
```
Comment (bspellacy) [inline spec/models/photo_spec.rb:2]: Let's just drop this spec. It doesn't really test anything.

**Review by bspellacy — APPROVED** [review]
This looks good to me!

**Pull request merged**